HW23/MainScreen/MediaTab/MediaView.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  MediaView.swift
+//  HW23
+//
+//  Created by Anton on 26.11.2022.
+//
+
+import SwiftUI
+
+struct MediaView: View {
+    private let settings = Settings.Media.self
+    @State var isPresented = false
+    
+    var body: some View {
+        NavigationView {
+            VStack {
+                if !isPresented {
+                    MainMediaView()
+                        .padding()
+                } else {
+                    EditScreenView()
+                }
+            }
+            .navigationTitle(Settings.TabBarTitles.media)
+            .navigationBarItems(
+                trailing: Button(isPresented ? settings.editButton : settings.doneButton) {
```

Comment:
```suggestion
                trailing: Button(isPresented ? settings.doneButton : settings.editButton) {
```
Наверное ты перепутал тут немного

Reply:
Там изначально должно быть Править, а когда внутри находишься то нажимаешь Готово